app/views/share/twitters/_share_fields.html.slim

```diff
@@ -1,3 +1,20 @@
-.form-group
+.form-group.twitter-share
   = label_with_tooltip(form, :description, t('share.twitter.form.description'), t('tooltips.shares.link_explanation'))
-  = form.text_area :description, class: 'form-control', maxlength: 124
+  = form.text_area :description, class: 'form-control', maxlength: 264
+  p.char-count <span> x </span> of 264
+
+javascript:
+  $(function() {
+    function updateTweetCount(textarea) {
+      var $span = $(textarea).next('p.char-count').find('span');
+      $span.text($(textarea).val().length);
```

Comment:
Worth noting that any links in the text will be altered to [23 characters](https://help.twitter.com/en/using-twitter/how-to-tweet-a-link):

> A URL of any length will be altered to 23 characters, even if the link itself is less than 23 characters long. Your character count will reflect this.

So the tweet count might only be useful when there are no urls in the tweet.

Reply:
Relevant: https://github.com/twitter/twitter-text/blob/master/js/src/getTweetLength.js

Reply:
@rodrei I handled this problem on the twitter storm builder gui - https://github.com/SumOfUs/toolkit/blob/master/src/TweetTool.js#L54

It uses https://github.com/twitter/twitter-text/tree/master/js to give you an accurate character length.

Reply:
Thanks for the feedback, I'll look at this later!